### PR TITLE
Added Kafka Connect JDBC Source SQL server

### LINKED ISSUE #425: Migrating Kafka Connect JDBC source SQL server
# What should we add to our content collections?
Migrate https://help.aiven.io/en/articles/2929789-kafka-connect-jdbc-source-with-sql-server

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Fixes #425

### CODE REVIEW

**Review by laysauchoa — APPROVED** [review]
Honestly, I like it! I do not have any more suggestions or improvements :)

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        "connection.password":"mypassword123",
+        "table.whitelist":"students,exams",
+        "mode":"incrementing",
+        "incrementing.column.name":"id",
+        "topic.prefix":"jdbc_source_sqlserver_increment.",
+        "tasks.max":"1",
+        "poll.interval.ms":"5000"
+    }
+
+With the above configuration stored in a ``jdbc_incremental_source_sqlserver.json`` file, you can create the connector in the ``demo-kafka`` instance with:
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:117]:
```suggestion
With the above configuration stored in the ``jdbc_incremental_source_sqlserver.json`` file, you can create the connector in the ``demo-kafka`` instance with:
```

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``SQLSERVER_HOST``, ``SQLSERVER_PORT``, ``SQLSERVER_DATABASE_NAME``, ``SQLSERVER_USER``, ``SQLSERVER_PASSWORD`` and ``SQLSERVER_TABLES``: source database parameters collected in the :ref:`prerequisite <connect_jdbc_sqlserver_source_prereq>` phase. 
+* ``mode``: the query mode, more information in the :doc:`dedicated page <../concepts/jdbc-source-modes>`; depending on the selected mode, additional configuration entries might be required.
+* ``topic.prefix``: the prefix that will be used for topic names. The resulting topic name will be the concatenation of the ``topic.prefix`` and the schema and table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default is 1, the connector can use at max 1 task for each source table defined.
+* ``poll.interval.ms``: query frequency, default 5000 milliseconds
+
+Check out the `dedicated documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector-config-options.rst>`_ for the full list of parameters.
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:64]: This seems important. I wonder if you think it makes sense to be as a note so the user can easily see somehow.

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:64]: I believe we have the same text in the three JDBC source example (PG, Mysql and sql-server) I would keep it as it is

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Example: define a JDBC incremental connector
+--------------------------------------------
+
+The example creates an :doc:`incremental <../concepts/jdbc-source-modes>` JDBC connector with the following properties:
+
+* connector name: ``jdbc_source_sqlserver_increment``
+* source tables: ``students`` and ``exams``, available in an Aiven for  Server database 
+* :doc:`incremental column name <../concepts/jdbc-source-modes>`: ``id``
+* topic prefix: ``jdbc_source_sqlserver_increment.``
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:95]:
```suggestion
* topic prefix: ``jdbc_source_sqlserver_increment``
```

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+    avn service connector create SERVICE_NAME @jdbc_source_sqlserver.json
+
+Check the connector status with the following command, replacing the ``SERVICE_NAME`` with the Aiven service and the ``CONNECTOR_NAME`` with the name of the connector defined before:
+
+::
+
+    avn service connector status SERVICE_NAME CONNECTOR_NAME
+
+Verify in the Apache Kafka target instance, the presence of the topic and the data
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:81]:
```suggestion
Verify in the Apache Kafka target instance, the presence of the topic and the data.
```

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        "topic.prefix":"KAFKA_TOPIC_PREFIX",
+        "tasks.max":"NR_TASKS",
+        "poll.interval.ms":"POLL_INTERVAL"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``SQLSERVER_HOST``, ``SQLSERVER_PORT``, ``SQLSERVER_DATABASE_NAME``, ``SQLSERVER_USER``, ``SQLSERVER_PASSWORD`` and ``SQLSERVER_TABLES``: source database parameters collected in the :ref:`prerequisite <connect_jdbc_sqlserver_source_prereq>` phase. 
+* ``mode``: the query mode, more information in the :doc:`dedicated page <../concepts/jdbc-source-modes>`; depending on the selected mode, additional configuration entries might be required.
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:59]: I would say the same here for consistency. I'd suggest either to keep a punctuation on the end or not but for all. Does this make sense?

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst:59]: fixed

**Comment by laysauchoa** [comment]
LGTM @ftisiot so from my side can be merged!

**Pull request merged**